stdlib/cache/src/main/ballerina/src/cache/Module.md

```diff
@@ -77,22 +82,22 @@ type CacheEntry record {|
 |};
 ```
 
-The linked list data structure is purely used for the eviction of the cache. According to the user configured eviction policy, when inserting / updating / retrieving cache entries, the linked list data structure should get updated. Therefore, when eviction happens, cache entries can be removed efficiently without iterating the complete map data structure.
+A linked list is used for the eviction of the cache. According to the user configured eviction policy, when inserting / updating / retrieving cache entries, the linked list will be updated. Therefore, when an eviction happens, cache entries can be removed efficiently without iterating the complete map data structure.
 
-**Example:** If the eviction policy is LRU, always the MRU item will be the head of the linked list. When eviction happens, nodes from the tail will be deleted without iterating the map.
+**Example:** If the eviction policy is LRU, always the MRU item will be the head of the linked list. When an eviction happens, nodes from the tail will be deleted without iterating the map.
 
-Further, developers can implement custom caching implementations based on different cache storage mechanisms (file, database. etc.) and different eviction policies (MRU, FIFO, etc.). Ballerina provides a "map-based cache" as the default cache implementation.
+Furthermore, the developers can implement custom caching implementations based on different cache storage mechanisms (file, database. etc.) and different eviction policies (MRU, FIFO, etc.). Ballerina provides a "map-based cache" as the default cache implementation.
 
 ### Samples
```

Comment:
We can remove the sample and add the BBE link instead of that .

Reply:
Sure.